server/utils/utils.ts

```diff
@@ -32,3 +32,7 @@ export const formatDate = (dateToFormat: string, dateFormat = 'd MMMM yyyy'): st
     return null
   }
 }
+
+export const pluralise = (word: string, count: string | number, plural = `${word}s`): string => {
```

Comment:
Moving this `pluralise` function to `utils` so it can be called both as a Nunjucks filter and used in rest of the code.